src/javascript/app_2/App/Components/Elements/popconfirm.jsx

```diff
@@ -0,0 +1,88 @@
+import React        from 'react';
+import classNames   from 'classnames';
+import PropTypes    from 'prop-types';
+import { localize } from '../../../../_common/localize';
+
+class PopConfirm extends React.Component {
```

Comment:
@zakibinary 
it's better to use PureComponent instead of Component when you don't need to handle `shouldComponentUpdate` by yourself.